src/System.Collections/tests/Generic/HashSet/HashSet.Generic.Tests.cs

```diff
@@ -335,6 +335,35 @@ public void SetComparer_SequenceEqualTests()
 
         #endregion
 
+        #region TryGetValue
+
+        [Fact]
+        public void HashSet_Generic_TryGetValue_Contains()
+        {
+            T value = CreateT(1);
```

Comment:
My bad. I see now that inheritance was already being used to make sure that that happens, so this is already done.